### PR TITLE
V3.511 inline sidebar

### PR DESCRIPTION
**Pull request by YauheniKapliarchuk**
Hi,

I need your review for this component.

@designerbrent 

1. Can you give me some instruction for design of this component in Editor?
2. Can I fix issues with close icon in the button?

Screen Shot 2020-12-02 at 20 18 58
Screen Shot 2020-12-01 at 21 31 14
Screen Shot 2020-12-01 at 21 31 19

### CODE REVIEW

File: src/blocks/inline-sidebar/components/edit.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+    render () {
+
+        const { className, attributes } = this.props;
+
+        return (
+            <div className={ className }>
+                <div className={"inline-sidebar"}>
+                    <fieldset>
+                        <legend>{ __("Author") }</legend>
```
Comment (designerbrent) [inline src/blocks/inline-sidebar/components/edit.js]: Change `Author` to `Preface`

File: src/blocks/inline-sidebar/components/edit.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                        <RichText
+                            className={"inline-sidebar__input"}
+                            placeholder={ __("Martin Luther King Jr.") }
```
Comment (designerbrent) [inline src/blocks/inline-sidebar/components/edit.js]: Replace `Martin Luther King Jr.` with `Enter a title preface.`

File: src/blocks/inline-sidebar/components/edit.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                            value={ attributes.author }
+                            onChange={ this.onChangeAuthor }
+                        />
+                    </fieldset>
+
+                    <fieldset>
+                        <legend>{ __("Title") }</legend>
+                        <RichText
+                            className={"inline-sidebar__input"}
+                            placeholder={ __("The Autobiography of Martin Luther King, Jr.") }
```
Comment (designerbrent) [inline src/blocks/inline-sidebar/components/edit.js]: Replace `The Autobiography of Martin Luther King, Jr.` with `Enter your block title`

**Review by designerbrent — CHANGES_REQUESTED** [review]
Please make the following changes: 

- [x] Rename the `Author` field. 
- [x] Fix the placeholder text.
- [x] Make the text in the `Description` allow for paragraphs and styling. Currently, it shows HTML tags on the front end for _bold_, _italic_, and _links_.
- [ ] Can we include **lists** and **headings** in the contents of the inline sidebar?

File: src/blocks/inline-sidebar/components/edit.js (hunk 1 above)
Comment (YauheniKapliarchuk) [reply src/blocks/inline-sidebar/components/edit.js]: It fixed

File: src/blocks/inline-sidebar/components/edit.js (hunk 2 above)
Comment (YauheniKapliarchuk) [reply src/blocks/inline-sidebar/components/edit.js]: It was fixed.

File: src/blocks/inline-sidebar/components/edit.js (hunk 3 above)
Comment (YauheniKapliarchuk) [reply src/blocks/inline-sidebar/components/edit.js]: It was fixed

**Review by designerbrent — CHANGES_REQUESTED** [review]
Any word on getting the **lists** and **headings** to work?

The rest of the changes look good.

**Comment by YauheniKapliarchuk** [comment]
Hi, @designerbrent

I investigated question about **"Headings"**  and **"Lists"** for inline sidebar.

Currently functionality of WP doesn't support extending default **"Headings"** and **"Lists"** for our components.

I propose:

1. to implement just new **"Headings"**(a few days for this component) component (because implementation  **"Lists"** component need much effort) for our needs and we will wait when WP team update functionality for using default components;
2. leave everything as it is and do not spend time for this functionality.

Thanks

**Review by designerbrent — APPROVED** [review]

**Comment by designerbrent** [comment]
Let's go with option 2 and just leave it as is. We an come back to adding headings and lists later, if needed.

**Pull request merged**